bin/offline-cluster.sh

```diff
@@ -9,36 +9,60 @@ set -x
 
 ls $ANSIBLE_DIR/inventory/offline
 
+if [ -f "$ANSIBLE_DIR/inventory/offline/hosts.ini" ]
+then
+        INVENTORY_FILE="$ANSIBLE_DIR/inventory/offline/hosts.ini"
+else
+        if [ -f "$ANSIBLE_DIR/inventory/offline/inventory.yml" ]
```

Comment:
Have you considered to use `elif` here? This would reduce the nesting a bit.

Reply:
makes sense, changed